website/client/components/groups/questSidebarSection.vue

```diff
@@ -1,74 +1,65 @@
-      button.btn.btn-primary.reject(@click='questReject(group._id)') {{$t('reject')}}
-    .row.quest-active-section(v-if='!onPendingQuest && onActiveQuest')
-      .col-12.text-center
-        .quest-boss(:class="'quest_' + questData.key")
-        h3(v-once) {{ questData.text() }}
-        .quest-box
-          .collect-info(v-if='questData.collect')
-            .row(v-for='(value, key) in questData.collect')
-              .col-2
-                div(:class="'quest_' + questData.key + '_' + key")
-              .col-10
-                strong {{value.text()}}
-                .grey-progress-bar
-                  .collect-progress-bar(:style="{width: (group.quest.progress.collect[key] / value.count) * 100 + '%'}")
-                strong {{group.quest.progress.collect[key]}} / {{value.count}}
-            div.text-right {{parseFloat(user.party.quest.progress.collectedItems) || 0}} items found
-          .boss-info(v-if='questData.boss')
-            .row
+      .quest(:class='`inventory_quest_scroll_${questData.key}`')
+    .col-6.titles
+      strong {{ questData.text() }}
+      p {{acceptedCount}} / {{group.memberCount}}
+    .col-4
+      button.btn.btn-secondary(@click="openQuestDetails()") {{ $t('details') }}
+  .row.quest-active-section.quest-invite(v-if='user.party.quest && user.party.quest.RSVPNeeded')
+    span {{ $t('wouldYouParticipate') }}
+    button.btn.btn-primary.accept(@click='questAccept(group._id)') {{$t('accept')}}
+    button.btn.btn-primary.reject(@click='questReject(group._id)') {{$t('reject')}}
+  .row.quest-active-section(v-if='!onPendingQuest && onActiveQuest')
+    .col-12.text-center
+      .quest-boss(:class="'quest_' + questData.key")
+      h3(v-once) {{ questData.text() }}
+      .quest-box
+        .collect-info(v-if='questData.collect')
+          .row(v-for='(value, key) in questData.collect')
+            .col-2
+              div(:class="'quest_' + questData.key + '_' + key")
+            .col-10
+              strong {{value.text()}}
+              .grey-progress-bar
+                .collect-progress-bar(:style="{width: (group.quest.progress.collect[key] / value.count) * 100 + '%'}")
+              strong {{group.quest.progress.collect[key]}} / {{value.count}}
+          div.text-right {{parseFloat(user.party.quest.progress.collectedItems) || 0}} items found
+        .boss-info(v-if='questData.boss')
+          .row
+            .col-6
+              h4.float-left(v-once) {{ questData.boss.name() }}
+            .col-6
+              span.float-right(v-once) {{ $t('participantsTitle') }}
+          .row
+            .col-12
+              .grey-progress-bar
+                .boss-health-bar(:style="{width: (group.quest.progress.hp / questData.boss.hp) * 100 + '%'}")
+          .row.boss-details
+            .col-6
+              span.float-left
+                | {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }} / {{ parseFloat(questData.boss.hp).toFixed(2) }}
```

Comment:
could those calculations be extracted into `data` or a method, that way its easier to move them around (on possible layout changes in the future)